### PR TITLE
Add a new rule creation guide

### LINKED ISSUE #75: Add small guide on how to create new rules
* How to use/write visitor
* How to parse per-rule configs (when implemented)
* How to implement BaseRule interface

### PR DESCRIPTION
**Pull request by incendial**
### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix
[ ] New rule
[ ] Changes an existing rule
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### What changes did you make? (Give an overview)
Closes #75 , Updates #147

### CODE REVIEW

**Comment by incendial** [comment]
@dkrutskikh @Anrock what do u think? Seems enough to me, but i'am a bit biased.

**Comment by Anrock** [comment]
@incendial LGTM. I don't think we need separate doc file for now. Maybe later when there is more docs on contributing.

**Review by Anrock — APPROVED** [review]

**Comment by Anrock** [comment]
Oh, maybe add a link / mention CONTRIBUTING file in readme also?

**Comment by incendial** [comment]
@Anrock done

**Review by Anrock — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        return visitor.binaryExpressions
+            .map((lit) => createIssue(
+                this,
+                _warningMessage,
+                '${lit.rightOperand} ${lit.operator} ${lit.leftOperand}',
+                _correctionComment,
+                source.url,
+                source.content,
+                source.compilationUnit.lineInfo,
+                lit))
```
Comment (dkrutskikh) [inline CONTRIBUTING.md:46]: @incendial missing trailing comma ?

**Review by dkrutskikh — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (incendial) [reply CONTRIBUTING.md:46]: it's an example, imo doesn't matter

**Pull request merged**